backends-velox/workload/tpch/gen_data/dwrf_dataset/tpch_convert_parquet_dwrf.sh

```diff
@@ -1,7 +1,8 @@
 batchsize=20480
 
 export GLUTEN_HOME=/PATH/TO/gluten/
-export GLUTEN_JVM_JAR=${GLUTEN_HOME}/backends-velox/target/gluten-1.0.0-SNAPSHOT-jar-with-dependencies.jar
+#please choose right os system jar
+export GLUTEN_JVM_JAR=${GLUTEN_HOME}/backends-velox/target/gluten-0.5.0-SNAPSHOT-jar-with-dependencies.jar
```

Comment:
also

Reply:
I think we don't need to use the accurate jar name, we can use `<gluten-jar>` instead, otherwise we must change the document and shell script each release.
We can just put the name format in one file, such as https://github.com/oap-project/gluten/blob/main/docs/GlutenUsage.md

Reply:
During Gluten's official release we probably try dynamical library loading based on detected arch / os at runtime.

For example:

gluten-xxx.jar
 |-centos-x86_64
   |-libxxx.so
   |-libxxx.so
 |-ubuntu-x86_64
   |-libxxx.so
 |-src
   |-main
     |-...

But still we may try our best to see if we can use the same native libraries for all linux releases.

Reply:
I agree, arrow-c-data is a good example, it distinguish os version n runtime